class-jan-9-20-complete.ipynb

```diff
@@ -89,7 +89,7 @@
     }
```

Comment:
<p>This seems fine, too.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jimstigler/jupyter/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>